examples/minimal/variables.tf

```diff
@@ -16,3 +16,20 @@ variable "tags" {
     Environment = "Example"
   }
 }
+
+variable "tamr_instance_tags" {
+  type        = map(string)
+  description = "A map of tags to add to the EC2 resource created by this example."
+  default = {
+    Author      = "Tamr"
+    Environment = "Example"
```

Comment:
Since we are adding tags, we could add something that reflects that this is an integration test. I think Tamr has a document with the standard tags